src/Microsoft.Performance.SDK/Processing/ProcessorOptions.cs

```diff
@@ -64,5 +66,27 @@ public class ProcessorOptions
         ///     passed to the processor.
         /// </summary>
         public IEnumerable<string> Arguments { get; }
+
+        public override bool Equals(object obj)
```

Comment:
Why do we need to override this and HashCode?

Reply:
We need to be able to compare our ProcessorOptions in tests to the expected options.

Reply:
I don't think that's true; see my comments on the tests.